tests/jsunit/db_test.js

```diff
@@ -1,76 +0,0 @@
-/**
```

Comment:
What happened to this file?

Reply:
It was not referenced in `index.html`, meaning none of the tests were being run.  It contained a single test (`test_DB_getNeighbours`) which failed when run.  Meanwhile `connection_db_test.js` contains a test of the same name which passed when run.  `db_test.js` appears to be dead code.